airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,253 @@
+version: 0.51.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - lists
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - '{{ parameters.field_path }}'
+  basic_authenticator:
+      type: BasicHttpAuthenticator
+      password: '{{ config.credentials.apikey }}'
+      username: 'anystring'
+  oauth_authenticator:
+      type: BearerAuthenticator
+      api_token: '{{ config.credentials.access_token }}'
+  requester:
+    type: CustomRequester
+    class_name: source_mailchimp.components.MailchimpRequester
+    http_method: GET
+    request_headers: {}
+    path: '{{ parameters.path }}'
+    request_parameters:
+      sort_dir: ASC
+      sort_field: '{{ parameters.cursor_field }}'
+    # authenticator:
+    #   class_name: source_mailchimp.components.MailchimpAuthenticator
+      # basic_auth: "#/definitions/basic_authenticator"
+      # oauth: "#/definitions/oauth_authenticator"
+    request_body_json: {}
+    basic_auth: "#/definitions/basic_authenticator"
+    oauth: "#/definitions/oauth_authenticator"
```

Comment:
The implementation is not correct.
Please check [this](https://github.com/airbytehq/airbyte/blob/54d40a9c3cbe79e21df5ec0b636f09909bf003ce/airbyte-integrations/connectors/source-retently/source_retently/manifest.yaml#L26-L33)